stress_test/controller.py

```diff
@@ -0,0 +1,349 @@
+# Copyright (c) 2016 Intracom S.A. Telecom Solutions. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v1.0 which accompanies this distribution,
+# and is available at http://www.eclipse.org/legal/epl-v10.html
+
+""" Controller Class- All controller-related functionality is here"""
+
+#import collections
+import json
+import common
+import logging
+import os
+import subprocess
+import time
+import util.netutil
+import util.process
+
+
+
+class Controller:
+
+
+    def __init__(self,ctrl_base_dir,test_config):
+
+#        logging.info('[Controller Class] Parsing test configuration')
+#        with open(json_file) as json_conf_file:
+#        self.test_config = json.load(json_conf_file)
+
+        """Create a Controller object. Options from JSON input file
+        :param test_config: JSON input configuration
+        :type test_config: JSON configuration dictionary
+        """
+        self.__dict__ = test_config
+        self.name = test_config['controller_name']
+        logging.info('Initializing parameters for the {0} Controller '.format(self.name))
+
+        #self.host_ip = test_config['controller_host_ip']
+        self.node_ip = test_config['controller_node_ip']
+        self.node_ssh_port = test_config['controller_node_ssh_port']
```

Comment:
lines 40-42: better name them `ssh_port`, `ssh_user`, `ssh_pass` (as in the onenote)